googletest/src/gtest.cc

```diff
@@ -800,41 +807,50 @@ std::string UnitTestImpl::CurrentOsStackTraceExceptTop(int skip_count) {
       );  // NOLINT
 }
 
+// returns the monotonic time in milliseconds
+TimeInMillis GetMonotonicTimeInMillis() {
+#if GTEST_LANG_CXX11
+  const std::chrono::steady_clock::duration now{ std::chrono::steady_clock::now().time_since_epoch() };
```

Comment:
This may probably work. Can't test it now though. Will have to wait to go back to work.